atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/OracleConnectionConfig.java

```diff
@@ -72,12 +74,17 @@ public String getUrl() {
     @Value.Derived
     @JsonIgnore
     public Optional<String> namespace() {
-        if (getSid().isPresent()) {
-            return getSid();
-        }
-        return serviceNameConfiguration().map(ServiceNameConfiguration::namespaceOverride);
+        return Stream.of(
```

Comment:
do we want to log (at least at debug, to match other logs in this class) if `namespaceOverride` is present?

Similarly, noting that check enforces exactly one of `sid` and `serviceNameConfiguration` to be present, which _may_ not be needed any more (although I don't have enough context to know if these params are required elsewhere).

Reply:
I think we still want the check - the purpose of `namespace` is mainly to outline how we identify ourselves to timelock (notice that we don't actually reference this field anywhere else in the DbKVS code). It doesn't actually see any use in connecting to the db

... in which case it probably makes sense for this not to be here.